conan/tools/microsoft/msbuilddeps.py

```diff
@@ -75,6 +80,13 @@ def __init__(self, conanfile):
                          'x86_64': 'x64'}.get(str(conanfile.settings.arch))
         # TODO: this is ugly, improve this
         self.output_path = os.getcwd()
+        # ca_exclude section
+        self.exclude_code_analysis = None
+        ca_exclude = self._conanfile.conf["tools.microsoft.msbuilddeps"].exclude_code_analysis
+        if ca_exclude is not None:
+            # TODO: Accept single strings, not lists
+            self.exclude_code_analysis = eval(ca_exclude)
```

Comment:
are we really okay if `eval` usage? it seems to be pretty dangerous, as it may allow arbitrary code execution:
https://nedbatchelder.com/blog/201206/eval_really_is_dangerous.html

Reply:
Yes, as well as every Conan recipe, how this would be more dangerous than Conan python evaluation?

Reply:
maybe we can avoid eval then? if use just a regexp, if regexp matches nothing, it acts effectively as disabled matching

Reply:
Problem is that all the matches we have in Conan for matching packages references are fnmatch based. Introducing now regex for this would be very asymmetric. And for fnmatch, we should probably allow multiple patterns, as it is not as expressive.

Reply:
Lets merge it, but keep it not documented while we finalize the UX, but there are some template changes that are useful for other PRs.